src/tabs/pill.scss

```diff
@@ -6,98 +6,54 @@
 
 @mixin iui-tabs-pill {
   @include iui-tabs;
-  display: flex;
   flex-flow: row;
   justify-content: space-evenly;
-  margin: $iui-baseline/2 0;
+  border-radius: $iui-border-radius;
+  gap: 2px;
+  @include themed {
+    background-color: t(iui-color-background-2);
+    border: 2px solid t(iui-color-background-2);
+  }
 
   > li {
-    margin: 0;
-    text-align: center;
     flex: 1 1 0;
 
-    > a {
-      line-height: $iui-line-height;
-      padding: ($iui-line-height * 0.5) 0;
-
-      @include themed {
-        background-color: t(iui-color-background-1);
-      }
+    > button {
+      height: $iui-baseline * 4;
+      background-color: transparent;
+      border-radius: $iui-border-radius;
+      width: 100%;
+      justify-content: center;
 
-      > svg {
+      &:enabled:hover {
```

Comment:
This has an extremely high specificity!

Reply:
Without the `:enabled` part, we will accidentally give the disabled tab a hover state.

I suppose an alternative would be to place this further down:
```
&[disabled]:hover {
    background-color: transparent;
}
```

Reply:
Yeah lets add/move all the disabled styles at the bottom

Reply:
I've removed all `:enabled` and styled `[disabled` at the bottom.